docs/concepts.rst

```diff
@@ -415,13 +415,20 @@ explictly specify a serialization strategy for an entity by attaching a
 
 .. _Protocol: api/protocols.rst
 
-Exporting Persisted Files
-.........................
+Retrieving Persisted Files
+..........................
+In some cases, you'll want to directly access the persisted file(s) for an
+entity rather than its in-memory representation.  (For example, if you're
+writing a paper or report, you may want to access the files containing the
+plots.) This can be achieved with the ``mode`` argument to
+:meth:`Flow.get <bionic.Flow.get>` method.  For example:
+
+.. code-block:: python
+
+    flow = builder.build().setting('subject', 'Alice')
+    flow.get('subject', mode=Path)
```

Comment:
nit: Do you think it's better to use `Path` or `'path'` in the example here?  I sliiiightly lean towards the latter but could go either way.

Reply:
I'm open to either, so `'path'` it is!